index.html

```diff
@@ -13,20 +13,66 @@
 
   <body>
     <header>
+      <img
+        src="./images/placeholder-avatar.png"
+        alt="Avatar Ulli"
+        class="header-image"
+      />
       <h1><em>Everything you need to know about me, Ulli</em></h1>
-      <nav>
-        <!-- change this to real section links with proper IDs -->
 
-        <a href="#section1">go to section 1</a>
-        <a href="#section2">go to section 2</a>
-        <a href="#section3">go to section 3</a>
-        <a href="#section4">go to section 4</a>
+      <nav>
+        
+        <a href="#me">about me</a>
+        <a href="#vaccines">about vaccines</a>
+        <a href="#section3">contact</a>
```

Comment:
You might want to point to your contact section:

```suggestion
        <a href="#contact">contact</a>
```